css/components/link.css

```diff
@@ -1,8 +1,19 @@
 .link-wrapper {
+  position: relative;
   display: flex;
   align-items: center;
```

Comment:
If the only content of `.link-wrapper` is `.link-content`, I think line 3 is superfluous, and line 4 belongs after line 8.